service/src/main/resources/application.yaml

```diff
@@ -138,15 +138,15 @@ appinsights:
 
 fpl:
   local_authority_email_to_code:
-    mapping: 'fpla.local=>FPLA;swansea.gov.uk=>SA;hillingdon.gov.uk=>HN;swindon.gov.uk=>SN;wiltshire.gov.uk=>SNW'
+    mapping: 'fpla.local=>FPLA;swansea.gov.uk=>SA;hillingdon.gov.uk=>HN;swindon.gov.uk=>SN;wiltshire.gov.uk=>SNW;croydon.gov.uk=>CR'
```

Comment:
I haven't seen new users added as part of this pr in add-users, do we need those changes?

Reply:
removed :) i think i was adding this here to test why the notification error was happening for raghu!